otx/algorithms/detection/adapters/mmdet/configurer.py

```diff
@@ -0,0 +1,757 @@
+"""Base configurer for mmdet config."""
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import importlib
+import json
+import os
+from typing import Any, Dict
+
+import numpy as np
+import torch
+from mmcv.runner import CheckpointLoader
+from mmcv.utils import Config, ConfigDict
+from torch import distributed as dist
+
+from otx.algorithms.common.adapters.mmcv.utils import (
+    align_data_config_with_recipe,
+    build_dataloader,
+    build_dataset,
+    patch_default_config,
+    patch_fp16,
+    patch_persistent_workers,
+    patch_runner,
+    update_basic_hooks,
+)
+from otx.algorithms.common.adapters.mmcv.utils.config_utils import (
+    recursively_update_cfg,
+    update_or_add_custom_hook,
+)
+from otx.algorithms.common.utils.logger import get_logger
+from otx.algorithms.detection.adapters.mmdet.utils import (
+    patch_datasets,
+    patch_evaluation,
+)
+
+logger = get_logger()
+
+
+# pylint: disable=too-many-public-methods
+class DetectionConfigManager:
```

Comment:
```suggestion
class DetectionConfigurer:
```

Reply:
If we use DetectionConfigManager, ClassficationConfigManager, etc, could this lead confusion with existed ConfigManger?

Reply:
That's the reason why I suggested to you to use `DetectionConfigurer`